features/step_definitions/common_steps.rb

```diff
@@ -25,11 +25,11 @@
   @current_user = User.find_by(email: VALID_EMAIL)
 end
 
-And "There are some events" do
+And "I have some events" do
```

Comment:
Kind of dangerous to re-use this setup 😓  Since it affects the testability of the homepage (View Upcoming Events feature). What if the system only displays events of this particular user on homepage, never another one?